### DISCUSSION TITLE
dlopen failed: library "libflutter_rust_bridge_example.so" not found

### DISCUSSION THREAD

**Discussion by andrew-ares** [fzyzcjy/flutter_rust_bridge#483] [Q&A] [answered]
But I got the lib already. What am I missing?

<img width="573" alt="1654422909494" src="https://user-images.githubusercontent.com/40336192/172045023-830b584c-d171-4c20-b517-2733c01d261e.png">

FYI: I'm using an emulator

**Comment by andrew-ares** [comment, marked as answer]
Problem solved. The fix is in here: http://cjycode.com/flutter_rust_bridge/template/setup_android.html#android_ndk-gradle-property.
Maybe we should add something here: http://cjycode.com/flutter_rust_bridge/tutorial_with_flutter.html#android-app?

**Reply by fzyzcjy** [reply]
Agree. Feel free to make a PR!
